.github/workflows/cargo-test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     env:
       RUSTFLAGS: -D warnings
       RUST_BACKTRACE: full
-      RUST_LOG: debug
+      RUST_LOG: warp_ipfs=error,warp_blink_wrtc=error
```

Comment:
erorrs are logged by default (as well as warnings) so this change essntially sets the log level to the default for **all** crates.

Reply:
To my knowledge, warnings arent logged by default, only errors (eg i usually see errors being logged when running test unless i explicitly set the variable to use a different level), however leaving it set to `RUST_LOG=debug` would log all underlining crates, which imo is not desirable to do on CI due to how much is outputted by all crates providing any logging.

Reply:
> To my knowledge, warnings arent logged by default, only errors (eg i usually see errors being logged when running test unless i explicitly set the variable to use a different level),

That's very strange. I always found it to be the other way around and that would make since warning is between info and error AFAIK. Maybe we're setting some default level in the code (implicitly or explicitly)? 🤔 

> however leaving it set to `RUST_LOG=debug` would log all underlining crates, which imo is not desirable to do on CI due to how much is outputted by all crates providing any logging.

Ah for sure. I agree with not logging all deps at debug level. However, our own crates should be set to debug at least IMO.